bittensor/utils/async_substrate_interface.py

```diff
@@ -1033,17 +1043,14 @@ def reload_type_registry(
         self, use_remote_preset: bool = True, auto_discover: bool = True
     ):
         """
-        Reload type registry and preset used to instantiate the SubtrateInterface object. Useful to periodically apply
-        changes in type definitions when a runtime upgrade occurred
-
-        Parameters
-        ----------
-        use_remote_preset: When True preset is downloaded from Github master, otherwise use files from local installed scalecodec package
-        auto_discover
-
-        Returns
-        -------
+        Reload type registry and preset used to instantiate the `AsyncSubstrateInterface` object. Useful to
+        periodically apply changes in type definitions when a runtime upgrade occurred
 
+        Args:
+            use_remote_preset: When True preset is downloaded from Github master, otherwise use files from local
+                               installed scalecodec package
```

Comment:
```suggestion
            use_remote_preset: When True preset is downloaded from Github master, otherwise use files from local installed scalecodec package
```